src/utils/messageBuilder.ts

```diff
@@ -0,0 +1,28 @@
+import { ItemType } from "../messages/message";
+import { Account } from "../accounts/account";
+
+type MessageBuilderConfig<T, C> = {
```

Comment:
Using `T` for `content` and `C` for `type` might be a bit misleading, the contrary would make much more sense

Reply:
Updated in [ea6951c](https://github.com/aleph-im/aleph-sdk-ts/pull/110/commits/ea6951cb62e03149743e73afbea87cf4d58e27de)